### PR TITLE
redirection fix on throwing exception

### PR DESCRIPTION
**Pull request by kastoestoramadus**

### CODE REVIEW

**Comment by kastoestoramadus** [comment]
With additional test we are blocked with another ticket. ~failing integration tests with real S3

**Comment by kastoestoramadus** [comment]
It can be tested on QA with providing bad S3 credentials.

**Comment by YuanZHANG910** [comment]
good with me, codes make sense and all tests passed.

File: test/uk/gov/hmrc/fileupload/controllers/RedirectionFeatureSpec.scala [hunk 1]
```diff
@@ -33,7 +33,7 @@ class RedirectionFeatureSpec extends UnitSpec with ScalaFutures with TestApplica
   private val allowedHosts = Seq[String]("gov.uk","localhost")
```
Comment (greg-gajda) [inline test/uk/gov/hmrc/fileupload/controllers/RedirectionFeatureSpec.scala]: Should you write some more UT to be sure that this new redirect handling works as expected?

Comment (kastoestoramadus) [reply test/uk/gov/hmrc/fileupload/controllers/RedirectionFeatureSpec.scala]: Yeah, I don't need to wait for IT tests to write a unit test.

**Pull request merged**